### ISSUE TITLE
New connections show own IP, not client IP

### ISSUE THREAD

**Issue by BrainStone** [mopidy/mopidy#1788]
When a new client connects, Mopidy logs the IP of itself rather than the new client.

Note that the IP of the server is `10.128.2.161` and client I connected from is `10.128.2.41`

Lines from the log:

- `/var/log/mopidy/mopidy.log`:  
  ```
  2019-08-20 08:31:57,539 INFO [12529:MpdSession-23] mopidy.mpd.session: New MPD connection from [::ffff:10.128.2.161]:6600
  ```
- `journalctl`:  
  ```
  Aug 20 08:45:40 CM-RASPI-1 mopidy[12529]: INFO     New MPD connection from [::ffff:10.128.2.161]:6600
  ```

**Comment by kingosticks** [comment]
Good catch, looks like we (I?) broke this in #1629.

**Comment by BrainStone** [comment]
Looks like this change is the culprit then:

https://github.com/mopidy/mopidy/pull/1629/files#diff-3d6996b56e7699dc4a4e56baf83555d8R28-R40

**Comment by kingosticks** [comment]
This was fixed in #1792 and will be available in the next release (v2.2.4).
